packages/snjs/lib/application.ts

```diff
@@ -1441,6 +1446,10 @@ public getSessions(): Promise<(HttpResponse & { data: RemoteSession[] }) | HttpR
     }
   }
 
+  public get settings(): SettingsProvider {
```

Comment:
The idea with Application is that it's the highest level interface for clients to interface with services. So it should not expose public properties like this. Instead it should expose the API of the services directly via high-level wrapper functions.

Reply:
Done. Resolving comment.